src/corner_kick/src/modules/Visualizer/components/Field.ts

```diff
@@ -0,0 +1,105 @@
+import * as PIXI from 'pixi.js';
+
+import { ROSConnector } from '~/ros/RosConnector';
+import { IFieldTopic } from '~/types/thunderbotsTopics';
+
+/**
+ * Displays a field in our visualizer
+ * Dimensions from the field are received from the ROS topic
+ * /backend/field
+ *
+ * TODO We are multiplying all values by 50 to make them bigger.
+ * Make this parameterizable and place it somewhere general.
```

Comment:
Could probably just put this in the constants file mentioned above

Reply:
Done